config.schema.json

```diff
@@ -16,6 +16,11 @@
         "type": "number",
         "required": true
       },
+      "key": {
+        "title": "Key",
```

Comment:
Can you make the title longer and more self-explanatory?  E.g. "API key for private sensors"

Reply:
sure thing! done